### PR TITLE
PostgreSQL updates

### PR DESCRIPTION
**Pull request by joesiewert**

### CODE REVIEW

**Comment by joesiewert** [comment]
Yeah, I flipped the order to go 10.3 to 9.3 instead of the other way around and it made for a messy looking diff, but the other ports should be the same.

**Comment by dennisnewel** [comment]
awesome - makes sense. Thanks!
Dennis Newel
Product Manager
Codeshipw:codeship.com  e:dennis.newel@codeship.com
t:@dennisnewel

Codeship API v2 is now in general availability. Learn more!  





On Mon, Mar 26, 2018 12:20 PM, Joe Siewert notifications@github.com  wrote:
Yeah, I flipped the order to go 10.3 to 9.3 instead of the other way around and
it made for a messy looking diff, but the other ports should be the same.

File: _basic/databases/postgresql.md [hunk 1]
```diff
@@ -23,90 +23,84 @@ categories:
...
 
-PostgreSQL 9.2 officially reached it's End-of-Life (EOL) (see [the PostgreSQL site for details](https://www.postgresql.org/support/versioning/)) which means that we will remove it from the Codeship Basic build image on **March 29, 2018**.
+PostgreSQL 10 is now the new default running on port 5432.
+</div>
 
-Since PostgreSQL 9.2 is currently running on port 5432, we will move PostgreSQL 10.3 from 5437 to 5432 to make 10.3 the new default. This should have minimal impact on users as PostgreSQL are generally very well backwards compatible, but we encourage you to try out 10.3 before March 29, 2018 to avoid any unexpected surprises.
+PostgreSQL `10` runs on the default port and the credentials are stored in the `PGUSER` and `PGPASSWORD` environment variables. The default databases created for you are **development** and **test**.
 
-### Python & earlier psycopg2 versions
+We install the Ubuntu `postgresql-contrib` package. It includes the [extension modules](https://www.postgresql.org/docs/9.6/static/contrib.html) listed in the PostgreSQL documentation.
```
Comment (mlocher) [inline _basic/databases/postgresql.md]: Please update the documentation link for 10, https://www.postgresql.org/docs/10/static/contrib.html

File: _basic/databases/postgresql.md [hunk 2]
```diff
@@ -23,90 +23,84 @@ categories:
...
 
-During testing, a very small number of users (2-4) experienced issues with older versions of the Python PostgreSQL client `psycopg2`. These older versions used a string-comparison to try and work out the PostgreSQL version number, and didn't account for a 10.x version. To resolve this issue, you'll need to upgrade psycopg2 to version 2.7 or higher.
+You need to activate them with `CREATE EXTENSION` as explained in the [Extension Guide](https://www.postgresql.org/docs/9.6/static/sql-createextension.html).
```
Comment (mlocher) [inline _basic/databases/postgresql.md]: https://www.postgresql.org/docs/10/static/sql-createextension.html

File: _basic/databases/postgresql.md (hunk 1 above)
Comment (joesiewert) [reply _basic/databases/postgresql.md]: ✅

File: _basic/databases/postgresql.md (hunk 2 above)
Comment (joesiewert) [reply _basic/databases/postgresql.md]: ✅

**Pull request merged**